### DISCUSSION TITLE
Pairing code / Setup Code problem

### DISCUSSION THREAD

**Discussion by G-D-D-971** [HomeSpan/HomeSpan#820] [Q&A]
I create a home accessory using Homespan... the problem is that if I change the configuration code and/or the setup code it becomes impossible to link my accessory, I tested by adding this code :
```
homeSpan.setPairingCode("26728027");
homeSpan.setQRID("JT92");
```
after the `void setup()` In example 4. Then I tried to remove my lines to reset the data with the CLI with the command `E` after `H`...
 The change of the pairing code / Setup code I have two possible results:
 - "The setup code is incorrect."
![317057552-6b1455b9-b074-46c9-9a5b-85db923c95e3](https://github.com/HomeSpan/HomeSpan/assets/124715754/8877f388-9339-4d07-9a2f-8047c6d985eb)

 - "Device is unreachable"

I use a Homespan 1.9.0

**Comment by HomeSpan** [comment]
This has already been covered.  Please see the issues list #786.
